ArcGISRuntimeSDKQt_QMLSamples/UtilityNetwork/DisplayUtilityAssociations/DisplayUtilityAssociations.qml

```diff
@@ -174,7 +179,7 @@ Rectangle {
                     SimpleLineSymbol {
                         id: attachmentSymbol
                         style: Enums.SimpleLineSymbolStyleDot
-                        color: "lime"
+                        color: "red"
```

Comment:
nice - assuming this fixes the UI bug Kathleen reported?

Reply:
Correct!